api/main.py

```diff
@@ -1,8 +1,10 @@
 from fastapi import FastAPI, Request, status
 from fastapi.encoders import jsonable_encoder
-from fastapi.exceptions import RequestValidationError
+from fastapi.exceptions import RequestValidationError, HTTPException
 from fastapi.responses import JSONResponse
 # from controller import Controller
+from starlette.responses import PlainTextResponse
```

Comment:
Why are we using starlette, instead of fast API responses - https://fastapi.tiangolo.com/tutorial/handling-errors/